runtime/doc/api.txt

```diff
@@ -822,6 +850,70 @@ nvim_buf_clear_highlight({buffer}, {src_id}, {line_start}, {line_end})
                     {line_end}    End of range of lines to clear (exclusive)
                                   or -1 to clear to end of file.
 
+                                                      *nvim_buf_lookup_mark()*
```

Comment:
Please write api function docs directly at the doc-string only. Everything at `api-global` and below will be auto-generated on by a script from the `api/*.c` docstrings, and must not be changed manually, lest it be overwritten by the script.

Reply:
Should i generate that as part of this pr?

Reply:
No, it will be done on master later.